### DISCUSSION TITLE
Apple Silicon builds?

### DISCUSSION THREAD

**Discussion by sezabass** [mockoon/mockoon#590] [Ideas] [answered]
It would be lovely to see a Mockoon release that is compatible with M1 chips.

**Comment by 255kb** [comment, marked as answer]
Indeed 😊
We are using electron-builder to package the application and I think they added support for M1 chips. So, I guess it would be relatively easy to add a new binary format.
However, as I don't own any Apple device, I would need help to test the binaries. Could you please open an issue to track this feature request? And would you be willing to do some testing for the next release?

**Reply by sezabass** [reply]
Nice! I read [here](https://www.electronjs.org/pt/blog/electron-11-0) that it does support Apple Silicon since v11.
And I'd be happy to help with the tests. Is it enought to follow [these guidelines](https://github.com/mockoon/mockoon/blob/main/CONTRIBUTING.md)?

**Reply by 255kb** [reply]
It's one thing that Electron supports new Apple silicon, but we also need [electron-builder](https://www.electron.build/) to be able to create compatible dmg files. I think that's the case, and we could just add the `arm64` `arch` in the `target` in the [electron-builder.json](https://github.com/mockoon/mockoon/blob/main/electron-builder.json) config file.

If you want to work and this you can, and yes the [guidelines](https://github.com/mockoon/mockoon/blob/main/CONTRIBUTING.md#maintainers-only-build-and-package-the-application-for-production) should be enough to package the application. Running `npm run package:mac` should create a `packages` folder with the binaries. 
Let me know if you just want to test, and I could build the binaries.
